ivy/functional/ivy/experimental/activations.py

```diff
@@ -291,8 +340,13 @@ def logsigmoid(
 @to_native_arrays_and_back
 @handle_array_function
 @handle_device_shifting
+@handle_complex_input
 def selu(
```

Comment:
Complex numbers are marked as unsupported for `selu` in the `tf`, `torch`, and `paddle` backends. The `paddle` backend has a custom implementation for `selu` on complex numbers, which should now be removed.

Reply:
`tf` doesn't have unsupported for complex numbers, `paddle` has been removed and test passed. `torch` on the otherhand throws an error that complex numbers are not supported. `Failed: [undefined]ivy.utils.exceptions.IvyBackendException: torch: selu: torch: elu: "elu_cpu" not implemented for 'ComplexFloat'`

Reply:
Done